samples/dashboard/demos/climate/demo.js

```diff
@@ -157,48 +275,13 @@ function createDashboard(worldMapJSON) {
                 }
             }],
             title: 'Selection Grid'
-        }, {
-            cell: 'kpi-chart',
-            type: 'Highcharts',
-            chartOptions: {
-                chart: {
-                    type: 'line'
-                },
-                series: [{
-                    name: 'Series from options',
-                    data: [1, 2, 1, 4]
-                }]
-            }
-        }, {
-            cell: 'world-map',
-            type: 'Highcharts',
-            chartConstructor: 'mapChart',
-            chartOptions: {
-                series: [{
-                    type: 'map',
-                    name: 'World Map',
-                    mapData: Highcharts.geojson(worldMapJSON)
-                }],
-                chart: {
-                    map: worldMapJSON,
-                    type: 'line'
-                },
-                title: {
-                    text: 'World Map'
-                }
-            },
-            events: {
-                mount: function () {
-                    // call action
-                    console.log('map mount event');
-                },
-                unmount: function () {
-                    console.log('map unmount event');
-                }
-            }
         }],
         editMode: {
-            enabled: false
+            enabled: true,
```

Comment:
Do we even need `editMode` in this demo?

Reply:
Moved to Asana